CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ if( GRPPI_OMP_ENABLE )
 endif( GRPPI_OMP_ENABLE )
 
 #FastFlow library
-option( GRPPI_FF_ENABLE "Require FastFlow library" ON )
+option( GRPPI_FF_ENABLE "Require FastFlow library" OFF )
```

Comment:
This change is not related to this issue. We should leave it to ON.

Reply:
That is right it slipped into my commit accidentally.